temporal-sdk/src/main/java/io/temporal/internal/client/external/GenericWorkflowClientImpl.java

```diff
@@ -386,4 +386,15 @@ public GetWorkerTaskReachabilityResponse GetWorkerTaskReachability(
                 .getWorkerTaskReachability(req),
         grpcRetryerOptions);
   }
+
+  @Override
+  public ExecuteMultiOperationResponse executeMultiOperation(ExecuteMultiOperationRequest req) {
```

Comment:
`executeMultiOperation` is a potentially long operation correct because we won't return until the update reaches the desired stage. If so we need to take a deadline here like the `update` call.

Reply:
Update is throwing a `WorkflowUpdateTimeoutOrCancelledException` upon timeout; I'm not quite sure if I should re-use it here or not. What do you think?